### PR TITLE
observables: extended LB observables for usage with CPU LB.

### PR DESCRIPTION
**Pull request by KaiSzuttor**
Description of changes:
 - Cylindrical LB profile extended for CPU LB plus test

### CODE REVIEW

**Comment by KaiSzuttor** [comment]
@fweik ready for review

File: testsuite/observable_cylindricalLB.py [hunk 1]
```diff
@@ -79,6 +76,7 @@ def set_particles(self):
         self.system.part.add(pos=self.positions)
 
     def set_fluid_velocity(self):
+        del self.positions[:]
```
Comment (fweik) [inline testsuite/observable_cylindricalLB.py:79]: What is that supposed to do?

Comment (KaiSzuttor) [reply testsuite/observable_cylindricalLB.py:79]: it removes positions stored as a member

Comment (fweik) [reply testsuite/observable_cylindricalLB.py:79]: Ah sorry, i thought this was on the system. Never mind.

**Comment by fweik** [comment]
@KaiSzuttor this can be merged even without the new lb interpolation code right?

**Comment by KaiSzuttor** [comment]
With the restriction that it only works for 1 node in case of CPU LB, id rather wait till we have a working function for getting the interpolated velocity on multiple nodes

**Comment by KaiSzuttor** [comment]
I think we should merge this. The fixes for the observables are quite important.

**Pull request merged**